grisera_api/graph_api_service.py

```diff
@@ -3,6 +3,60 @@
 from pydantic import BaseModel
 
 
+def create_properties_from_list_of_dict(list_of_dicts: list):
+    """
+    Creates request body from list of dictionaries with properties
+
+    Args:
+        list_of_dicts (dict): list of dictionaries of properties
+
+    Returns:
+        Request body
+    """
+    request_body = []
+    for dict_property in list_of_dicts:
+        keys = list(dict_property.keys())
+        values = list(dict_property.values())
+        for k, v in zip(keys, values):
+            request_body.append({"key": k, "value": v})
+    return request_body
```

Comment:
I think using `items()` would make it a little bit more readable:
```
    request_body = []
    for dict_property in list_of_dicts:
        for k, v in dict_property.items():
            request_body.append({"key": k, "value": v})

    return request_body
```
We can go even further using list and dict comprehension combined:
```
    request_body = [{"key": k, "value": v}
                    for dict_property in list_of_dicts for k, v in dict_property.items()]
    return request_body
```
Although I've some doubts if it's still readable :wink: